Organization.md

```diff
@@ -0,0 +1,102 @@
+# Organization
+
+Currently, the Distributed Text Services is not legally formalized as an organization (outside of the Github Organization and the Google Group).
+
+Although, the Distributed Text Services specifications is lead by a [Technical Committee](#technical_committee). It is responsible for the evolution and documentation of the specifications.
```

Comment:
Recommend changing this to:

The Distributed Text Services is a organized and led by a community of volunteers. It is not currently a legal entity.  Decisions are made collectively by the [Technical Committee](#technical_committee).  Members of this committee share ownership of the GitHub repository and the Google Group. The committee also is responsible for the evolution and documentation of the specification itself.